homeassistant/components/switch/broadlink.py

```diff
@@ -245,8 +245,10 @@ def _sendpacket(self, packet, retry=2):
     def _auth(self, retry=2):
         try:
             auth = self._device.auth()
-        except socket.timeout:
+        except socket.timeout as error:
             auth = False
+            if retry < 1:
+                _LOGGER.error(error)
```

Comment:
I think it would be better to change the log to something like: "Timed out during authorization"